### DISCUSSION TITLE
Darstellungsfilter bei Ergebnis

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#217] [General]
Hallo,
im neuen "Professional" Layout ist der Darstellungsfilter für ein einzelnes Ergebnis "verschwunden". 
![image](https://github.com/user-attachments/assets/6c03d26e-8087-4c36-befd-91f23c0cabee)
![image](https://github.com/user-attachments/assets/02d4888f-7753-4668-b04a-ea8df5516978)

Und auch die "webgis.usability.singleResultButtons.push"-Tools sind im neuen Layout nicht mehr auffindbar.

Auch wenn das Ergebnis selektiert wird, erscheinen keine zusätzlichen Tools.

Ist das Absicht?

lg

**Comment by gis-eni** [comment]
Das Problem ist hie natürlich wir hin mit diesen Werkzeugen. Die Tools die oben angezeigt werden, beziehen sich bei einer Tabellen Ansicht auf alle Zeilen. Die Tools bei den "alten" einzel-Ergebnissen beziehen sich auf ein einzelnes Ergebnis. Man könnte jetzt diese Werkzeuge in jeder Zeile der Tabelle als Button anbieten, das würde halt das Layout sprengen und bei vielen Buttons nicht mehr gut aussehen.

Nachdem es "Professional" Layout eigentlich ein Layout für Desktop User ist, die mit der Maus arbeiten, war die Lösung vor, diese auf die Rechte Maustaste zu legen:

![image](https://github.com/user-attachments/assets/980964fc-5db8-479f-8699-244e59106911)

Dort werden alle Tools angezeigt, die sich nur auf auf diese Zeile beziehen. Neben den Werkzeugen werden hier auch gleich noch einmal alle Hotlinks aufgelistet.

Der Hinweis, für diese Vorgehensweise steht übrigens schon in deiner Frage ;)

![image](https://github.com/user-attachments/assets/fc8cbc36-73ee-40d8-9388-854279dd30b0)

**Reply by marko-jer** [reply]
Jaja, wer lesen kann, ist klar im Vorteil ;)

Die Lösung mit rechter Maustaste ist OK!
